wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1347,25 +1347,26 @@ def keyboard_shortcuts_dialog(context):
                     f"{KEYS.MOD} + {KEYS.SHIFT} + z" if is_mac else f"{KEYS.MOD} + y",
                 ),
             ],
-            ("actions-model", _("Actions")): [
-                (_("Show keyboard shortcuts"), "?"),
+            ("actions", _("Actions")): [
                 (_("Save changes"), f"{KEYS.MOD} + s"),
                 (_("Preview"), f"{KEYS.MOD} + p"),
-                (_("Toggle sidebar"), "["),
-                (_("Toggle minimap"), "]"),
-                (_("Search"), "/"),
                 (_("Add or show comments"), f"{KEYS.CTRL} + {KEYS.ALT} + m")
                 if comments_enabled
                 else None,
             ],
-            ("rich-text-content", _("Text content")): [
-                (_("Insert or edit a link"), f"{KEYS.MOD} + k")
+            ("application", _("Application")): [
+                (_("Show keyboard shortcuts"), "?"),
+                (_("Search"), "/"),
+                (_("Toggle sidebar"), "["),
+                (_("Toggle minimap"), "]"),
+                (_("Close modal dialogs (like this one)"), f"{KEYS.ESC}"),
             ],
-            ("rich-text-formatting", _("Text formatting")): [
+            ("text-formatting", _("Text formatting")): [
+                (_("Bold"), f"{KEYS.MOD} + b"),
                 (_("Italic"), f"{KEYS.MOD} + i"),
-                (_("Underline"), f"{KEYS.MOD} + u"),
+                (_("Insert or edit a link"), f"{KEYS.MOD} + k"),
                 (_("Monospace (code)"), f"{KEYS.MOD} + j"),
-                (_("Strike-through"), f"{KEYS.MOD} + x"),
+                (_("Strike-through"), f"{KEYS.MOD}+ {KEYS.SHIFT} + x"),
```

Comment:
```suggestion
                (_("Strike-through"), f"{KEYS.MOD} + {KEYS.SHIFT} + x"),
```

White space missing.